src/test/features/response/form/models/timelineRow.ts

```diff
@@ -0,0 +1,57 @@
+import { expect } from 'chai'
+
+import { TimelineRow, ValidationErrors } from 'response/form/models/timelineRow'
+import { Validator } from 'class-validator'
+import { expectValidationError } from '../../../../app/forms/models/validationUtils'
+
+describe('TimelineRow', () => {
+
+  describe('empty', () => {
+
+    it('should return empty instances of TimelineRow', () => {
+
+      const actual: TimelineRow = TimelineRow.empty()
+
+      expect(actual instanceof TimelineRow).to.eq(true)
```

Comment:
You can use [`instanceOf`](http://chaijs.com/api/bdd/#method_instanceof) Chai assertion here.